2-mandatory/1-create-functions.js

```diff
@@ -24,7 +28,8 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(arr) {
+  return arr.map((String) => String.trim().replace("/","").toLowerCase());
```

Comment:
careful of your naming here; variables in javascript are `lowerCamelCase` by convention, and the `String` name is actually a key word for the `String` class in javascript so should not be used as a variable name.